src/RedisProxy.php

```diff
@@ -300,10 +309,11 @@ public function pexpire($key, $miliseconds)
     /**
      * Set the expiration for a key as a UNIX timestamp
      * @param string $key
-     * @param int $timestamp
+     * @param int    $timestamp
```

Comment:
tieto medzery by si mi sem nemusel davat :D

Reply:
prebehnem to nasim formatterom